### DISCUSSION TITLE
Multi-vendor apps

### DISCUSSION THREAD

**Discussion by sdekna** [ever-co/ever#1510] [General]
Hi all,

I see that system is multi-vendor based, however, the apps include single vendor apps... How would I be able to make it multi-vendor shop app?

Many thanks.

**Comment by evereq** [comment]
Hello @sdekna , the mobile shop app is multi-vendor because it displays products from different stories close to customer locations.

**Reply by sdekna** [reply]
can it display vendors in thr vicinity then show products of the vendors? like uber eats for example?

**Reply by evereq** [reply]
Short answer - yes, but I think you will need to build some custom pages for that.  By default app just display a list of products, not vendors. 

Note: we do have a feature where we display vendors near user location but it's more for the "in-store" experience when someone wants to go inside the store and buy products there. But that feature is different etc.
